executor/show.go

```diff
@@ -732,7 +739,7 @@ func ConstructResultOfShowCreateTable(ctx sessionctx.Context, tableInfo *model.T
 			fmt.Fprintf(buf, " COMMENT '%s'", format.OutputFormat(col.Comment))
 		}
 		if i != len(tableInfo.Cols())-1 {
-			buf.WriteString(",\n")
+			needAddComma = true
```

Comment:
If the last 2 columns are hidden, the `show create table` statement will result like (the table has 3 columns, `b` and `c` are hidden columns): 
```
CREATE TABLE `hidden` (
  `a` int(11) NOT NULL,
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin")
```

Reply:
I check the `hidden` before writing `,`   so It won't happen.